data/json/items/generic/bedding.json

```diff
@@ -1,4 +1,22 @@
 [
+  {
+    "id": "pillowcase",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "pillowcase" },
+    "looks_like": "pillow",
+    "description": "A simple cotton pillowcase.",
+    "weight": "415 g",
+    "volume": "1 L",
+    "price": 0,
+    "price_postapoc": 2,
+    "to_hit": -5,
+    "material": [ "cotton" ],
+    "flags": [ "SLEEP_AID_CONTAINER" ],
+    "pocket_data": [ { "pocket_type": "CONTAINER", "max_contains_volume": "15 L", "max_contains_weight": "15 kg", "moves": 400 } ],
```

Comment:
Pillows are currently ~1L so 15L seems too large for a pillow case.

Reply:
This should probably go on the other PR.